<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>19</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Primer Grupo de Operaciones</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="opcion4">
       <property name="text">
        <string>Opcion 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="opcion5">
       <property name="text">
        <string>Opcion 2</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Segundo Grupo de Operaciones</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="opcion6">
       <property name="text">
        <string>Opcion 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="opcion7">
       <property name="text">
        <string>Opcion 4</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="opcion1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>149</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Opcion 1</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="opcion2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>149</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Opcion2</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="opcion3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>149</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Opcion 3</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="boton1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Habilitar 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Habilitar 1</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="valores">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>378</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
